qt/html_processor.cpp

```diff
@@ -0,0 +1,51 @@
+#include "html_processor.hpp"
+
+#include <string>
+
+#include <QString>
+
+QString ProcessCopyrightHtml(std::string const & html, std::string const & lang)
+{
+  QString result = html.c_str();
```

Comment:
Can conversion to QString be avoided and unnecessary string copies be avoided by using std::string_view? If yes, then this method can be generic and exclude lang tags even without Qt dependency in other places/platforms.

Reply:
On another side, if it is only related to the About dialog now, and specific processing of copyright.html with only p tags supported, then it would be cleaner to put this function into about.cpp, where it is used.

Reply:
Right!